diff --git a/api/smtpd-defines.h b/api/smtpd-defines.h
index b73eefe..418e6dd 100644
--- a/api/smtpd-defines.h
+++ b/api/smtpd-defines.h
@@ -16,6 +16,9 @@
  * OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
  */
 
+#ifndef _SMTPD_DEFINES_H
+#define _SMTPD_DEFINES_H
+
 #ifndef nitems
 #define nitems(_a) (sizeof((_a)) / sizeof((_a)[0]))
 #endif
@@ -63,9 +66,10 @@ enum smtp_proc_type {
 #define	SMTPD_MAXHOSTNAMELEN	256
 #define	SMTPD_MAXLINESIZE	2048
 
-#define SMTPD_USER		"_smtpd"
 #define PATH_CHROOT		"/var/empty"
 #define SMTPD_QUEUE_USER	 "_smtpq"
 #define PATH_SPOOL		"/var/spool/smtpd"
 
 #define TAG_CHAR	'+'	/* gilles+tag@ */
+
+#endif
